client/flutter/lib/main.dart

```diff
@@ -5,15 +5,24 @@ import './home_page.dart';
 import './constants.dart';
 import 'package:flutter_localizations/flutter_localizations.dart';
 import 'generated/l10n.dart';
+import 'package:firebase_analytics/firebase_analytics.dart';
+import 'package:firebase_analytics/observer.dart';
 
 void main() => runApp(MyApp());
 
 class MyApp extends StatefulWidget {
+  static FirebaseAnalytics analytics = FirebaseAnalytics();
```

Comment:
Where do we actually configure the firebase app id?

Reply:
OP mentions that it needs to be in `goog-services.json`.